static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -63,11 +64,17 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
     icon: <IconLocation size="xs" />,
   }),
   issue: (frame: ErrorFrame) => ({
-    color: 'red300',
-    description: frame.message,
+    color: frame.message === 'User Feedback' ? 'pink300' : 'red300',
+    description:
```

Comment:
we could make this description more interesting, by putting the message preview for example, but that would involve an extra API call to fetch the issue details

Reply:
Could we put the message into something like the `error.value` field of the response object?

the response to `/replays-events-meta/` is something like:
```
{
    "data": [
        {
            "issue.id": 5071000502,
            "error.type": "",
            "error.value": "",
            "id": "75deaeb8eb7940ed92d9612c49a1e39a",
            "timestamp": "2024-03-15T19:44:54+00:00",
            "title": "User Feedback",
            "project.name": "pokemart",
            "issue": "POKEMART-8P"
        }
    ],
    "meta": {
        "fields": {
            "issue.id": "integer",
            "error.type": "string",
            "error.value": "string",
            "id": "string",
            "timestamp": "date",
            "title": "string",
            "project.name": "string"
        },
        "units": {
            "issue.id": null,
            "error.type": null,
            "error.value": null,
            "id": null,
            "timestamp": null,
            "title": null,
            "project.name": null
        },
        "isMetricsData": false,
        "isMetricsExtractedData": false,
        "tips": {
            "query": null,
            "columns": null
        },
        "datasetReason": "unchanged"
    }
}
```

cc @JoshFerge

Reply:
Not easily. we could make a backlog item to address this though, maybe by giving user feedback its own endponit

Reply:
ack. kk